yetanotherco/aligned_layer
# test(explorer): set a max_batch_size to the explorer

**Motivation**

This PR is solely to test #1198 

**Description**

**How to Test**

1. Start anvil:

```
make anvil_start_with_block_time
```

2. Start the explorer

```
make explorer_build_db
make explorer_create_env
make run_explorer
```

3. Start the test:

```
cd operator/pkg
go test -run TestGetBatchExplorerOOM -v -timeout 0
```

In the explorer, you should be able to see the following output:
```
[error] Error calculating proof hashes: {:http, :body_too_large}
```
